### PR TITLE
fix: bump jquery-ui to v1.13.2 to fix multiple CVEs

### PR DESCRIPTION
**Pull request by daschnerm**
## What

- bump jquery-ui to v1.13.2 to fix CVE-2016-7103
- I've tested the floating ToC, as well as the normal ToC feature still works without any issues.

## Why

https://github.com/rstudio/rmarkdown/issues/2405

### CODE REVIEW

File: inst/rmd/h/jqueryui/jquery-ui.css [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+* Includes: draggable.css, core.css, resizable.css, selectable.css, sortable.css, accordion.css, autocomplete.css, menu.css, button.css, controlgroup.css, checkboxradio.css, dialog.css, progressbar.css, selectmenu.css, slider.css, spinner.css, tabs.css, tooltip.css, theme.css
```
Comment (cderv) [inline inst/rmd/h/jqueryui/jquery-ui.css:3]: Do you think we can just keep the component we had ? 

It seems that this adds controlgroup, checkboxradio . Are they needed ¿

**Review by cderv — COMMENTED** [review]
Thanks a lot. 

I left a comment. I wonder if we should have exactly the same component. 

Otherwise, I was wondering if we should sync with the version in shiny. What do you think ?  
I did compare with this script executed on your branch 
```r
# This script aims to update and sync jquery-ui dependency with the one in shiny

rmd <- "inst/rmd/h/jqueryui/"

shiny <- "https://github.com/rstudio/shiny/archive/refs/heads/main.zip"

temp_zip <- tempfile(fileext = ".zip")
xfun::download_file(shiny, temp_zip, mode = "wb")

dir.create(temp_shiny <- tempfile("shiny"))

unzip(temp_zip, exdir = temp_shiny)

jqueryui <- file.path(temp_shiny, "shiny-main", "inst", "www", "shared", "jqueryui")

unlink(rmd, recursive = TRUE)

file.copy(jqueryui, dirname(rmd), overwrite = TRUE, recursive = TRUE)

unlink(temp_shiny, recursive = TRUE)
unlink(temp_zip, recursive = TRUE)
````

It seems there are some differences probably due to the  components includes.

File: inst/rmd/h/jqueryui/jquery-ui.css (hunk 1 above)
Comment (daschnerm) [reply inst/rmd/h/jqueryui/jquery-ui.css:3]: I don't think that they are needed, but I didn't check the code. However the jquery-ui from Shiny includes both of them:

```
diff shiny-css.txt rmarkdown-main-css.txt 
4,5d3
< checkboxradio.css
< controlgroup.css
7d4
< datepicker.css
```

If we want to sync with shiny, then I guess we should use the same components ?

**Comment by daschnerm** [comment]
@cderv I've updated the PR and pulled in the`jquery-ui` version from Shiny.

**Review by cderv — APPROVED** [review]

**Pull request merged**